cra-server/.env_sample

```diff
@@ -7,4 +7,20 @@
 #    
 # In this example the first line configures a ui server called aaa with a remote server name of "cocoView1" and URL of "https://localhost:9443"
 EGERIA_PRESENTATIONSERVER_SERVER_aaa={"remoteServerName":"cocoView1","remoteURL":"https://localhost:9443"}
-EGERIA_PRESENTATIONSERVER_SERVER_bbb={"remoteServerName":"cocoView2","remoteURL":"https://localhost:9443"}
\ No newline at end of file
+EGERIA_PRESENTATIONSERVER_SERVER_bbb={"remoteServerName":"cocoView2","remoteURL":"https://localhost:9443"}
+
+# Environment variables used to secure communications between the presentation server and the view servers  
+
+# This is the ca option as defined in the node documentation https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options 
+EGERIA_PRESENTATIONSERVER_EGERIA_CA=ssl/truststore.p12
+# This is the pfx option as defined in the node documentation https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options  
+EGERIA_PRESENTATIONSERVER_EGERIA_PFX=ssl/keystore.p12
+# This is the passphrase option as defined in the node documentation https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options  
+EGERIA_PRESENTATIONSERVER_EGERIA_PFX_PASSPHRASE=egeria
+
+# Environment variables used to secure communications between the browser and the presentation server
+
+# This is the file location of the certificate used to secure communications between the browser and the presentation server
+EGERIA_PRESENTATIONSERVER_EGERIA_BROWSER_CERT=ssl/keys/server.cert
+# This is the location of the file containing the key used to secure communications between the browser and the presentation server
+EGERIA_PRESENTATIONSERVER_EGERIA_BROWSER_PASSPHRASE=ssl/keys/server.key
```

Comment:
The approach to setting the vars from the CLI needs clarifying. It's confusing currently.

I have a PR at https://github.com/odpi/egeria-react-ui/pull/84 which did this for the original var which I think is a slight improvement - but would need merging with these changes. Open to what order you merge in but I think that clarity is a must

Reply:
I have merged in 84, I had not pushed up all the code - sorry about that.